### DISCUSSION TITLE
[Features Request]

### DISCUSSION THREAD

**Discussion by Galaxy0Gamer** [rohitsangwan01/uni_control_hub#33] [Ideas]
The tool is so cool and I want it to be cooler.

**Features Requested:**

- Drag and drop, doesn't have to be a literal drag and drop, but maybe a way so that when I drag a file to the direction specified, an adb push command is made onto the phone.
- Clipboard sharing, would be cool to have this feature built-in.
- The ability to use special keys, like CTRL, ALT, etc. For some reason they don't work.
- Make the app automatically know the resolution and orientation of my phone, I don't know wherther this is possible or not, but maybe use an app on the phone to send info like the resolution and the orientation.

**Comment by rohitsangwan01** [comment]
Most of these features are on my roadmap, but there are few blockers, or last option would be to ship a mobile app as well

**Comment by julienR2** [comment]
> last option would be to ship a mobile app as well

Personally I absolutely love the fact that nothing is to be installed on the device.
In case it's required for some feature, it would be awesome to keep it optional (although I guess it could be tricky).

I take advantage from this message for a huge shout out for this app! First experience was flawless, and exactly what I was looking for 👏🎉

**Comment by rohitsangwan01** [comment]
@julienR2 Thank you, yeah few things requires a client side app on mobile, but will definitely try to keep that optional
